dashboard/modules/reporter/reporter_agent.py

```diff
@@ -77,7 +77,25 @@ def __init__(self, dashboard_agent):
             "node_cpu": Gauge("node_cpu", "Total CPU usage on a ray node",
                               "percentage", ["ip"]),
             "node_mem": Gauge("node_mem", "Total memory usage on a ray node",
-                              "mb", ["ip"]),
+                              "bytes", ["ip"]),
+            "node_disk_usage": Gauge("node_disk_usage",
+                                     "Total disk usage (bytes) on a ray node",
+                                     "bytes", ["ip"]),
+            "node_disk_utilization_percentage": Gauge(
+                "node_disk_utilization_percentage",
+                "Total disk utilization (percentage) on a ray node",
+                "percentage", ["ip"]),
+            "node_network_sent": Gauge("node_network_sent",
+                                       "Total network sent", "bytes", ["ip"]),
```

Comment:
bytes/s is a weird unit for network util. Usually you use bits (likely Kb in this case). If this is what we're showing in the dashboard, it's fine for now